client/my-sites/plugins/plugin-management-v2/plugin-row-formatter/index.tsx

```diff
@@ -92,7 +92,7 @@ export default function PluginRowFormatter( {
 	);
 
 	const { activation, autoupdate } = useSelector( ( state ) =>
-		getAllowedPluginActions( item, state, selectedSite )
+		getAllowedPluginActions( state, item, selectedSite )
```

Comment:
I updated the `getAllowedPluginActions` selector to follow selector conventions: `state` is the first argument.